core/src/main/java/io/questdb/griffin/update/UpdateStatement.java

```diff
@@ -56,10 +68,21 @@ public int getTableId() {
         return tableId;
     }
 
+    @Override
+    public int getTableNamePosition() {
+        return 7;
```

Comment:
I may be multiple spaces before table name, should not be a constant

Reply:
agreed, this needs to be differently, just hacked this in for now

Reply:
this method has been removed